src/common/types/enums/ecdsa.ts

```diff
@@ -0,0 +1,6 @@
+enum ECDSA {
```

Comment:
@intelliot @ledhed2222 I created an enum type for ECDSA methods for ecdsa-secp256k1 and ed25519. This replaces string literals used throughout our codebase to improve maintainability. Let me know if this enum name and values look good. 🙂